main.nf

```diff
@@ -573,6 +574,12 @@ process Compute_Connectivity_without_similiarity {
         --parcel_volume $labels $labels_list
     scil_normalize_connectivity.py vol.npy sc_vol_normalized.npy \
         --parcel_volume $labels $labels_list
+
+    #!/usr/bin/env python
+    import numpy as np
+    for data in ["sc", "vol", "len", "sc_edge_normalized", "sc_vol_normalized"]:
```

Comment:
I would do a for loop with os.listdir() and check if the extension is *.npy
The input allow for metrics that leads to any number of output npy